examples/counter/sources/counter.move

```diff
@@ -6,8 +6,7 @@ module rooch_examples::counter {
       value:u64,
    }
 
-   fun init(ctx: &mut StorageContext, account: &signer){
-      account_storage::global_move_to(ctx, account, Counter{value:0});
```

Comment:
Why remove the `init` function?

Reply:
Okay, I will put back the previously deleted `init` function.

Another question is... I'm not sure if it is possible to write like this in the move file under the integration-test directory:

```move
script {
    use moveos_std::storage_context::StorageContext;
    use rooch_examples::counter;

    fun main(ctx: &mut StorageContext, sender: &signer) {
        counter::init(ctx, sender);
    }
}
```

Is it possible to automatically fill in `ctx: &mut StorageContext` inside the script section?

Reply:
I compiled the Move code according to the example above, but I received an error during compilation:

```move
Error: error: undeclared `0x2::storage_context::StorageContext`
  ┌─ /tmp/tempfile:8:26
  │
8 │     fun test1(_ctx: &mut StorageContext, sender: &signer) {
  │                          ^^^^^^^^^^^^^^
```

I found the reason: in the source code of the Move Compiler, when compiling the `Script` into `ModuleEnv` using `ModelBuilder`, it cannot find `StorageContext` in `struct_table`.
This seems to be a problem, I will investigate further.

Reply:
> Okay, I will put back the previously deleted `init` function.
> 
> Another question is... I'm not sure if it is possible to write like this in the move file under the integration-test directory:
> 
> ```move
> script {
>     use moveos_std::storage_context::StorageContext;
>     use rooch_examples::counter;
> 
>     fun main(ctx: &mut StorageContext, sender: &signer) {
>         counter::init(ctx, sender);
>     }
> }
> ```
> 
> Is it possible to automatically fill in `ctx: &mut StorageContext` inside the script section?

The code should be worked.

Reply:
Maybe, we should publish the modules of the project first, then run the integration-test?

Reply:
The reason for this issue is that during the compilation of `private_generics`, `build_model` is required, but the compiled `FullyCompiledProgram` is not imported as a dependency, resulting in an error.

It is not related to compiling the `Move Package` as a dependency or the subsequent `integration-test` errors.

Furthermore, since `build_model` does not support directly importing `FullyCompiledProgram` as a dependency, the results of the `Expansion` type in `FullyCompiledProgram` need to be manually imported into `ModelBuilder.struct_table` to resolve the issue.

I suggest merging this PR first because currently, as long as types from `stdlib` are not imported in the Script (importing functions is supported), there won't be any issues.

Reply:
Because the `init` is private, we can not invoke it in the script. Or add another public `init` function for the integration tests to call.

Reply:
Or `init_for_test`

Reply:
In the file `examples/counter/integration-tests/counter.move`, there is a call to the function `counter::init_`.

https://github.com/rooch-network/rooch/blob/ca3ef47d749663e030e55bb5b4f0c38d7cba6a0e/examples/counter/integration-tests/counter.move#L1-L11

However, due to the reason mentioned earlier, we currently cannot directly import the `&mut StorageContext` type in the Script.